docs/sources/tempo/traceql/query-editor.md

```diff
@@ -11,20 +11,66 @@ keywords:
   - TraceQL
 ---
 
-# TraceQL query editor
+# Write TraceQL queries in Grafana
 
-You can use the TraceQL viewer and query editor in the Tempo data source to build queries and drill-down into result sets. The editor is available in Grafana’s Explore interface.
+You can compose TraceQL queries in Grafana and Grafana Cloud using **Explore** and a Tempo data source. You can use either the **Query type** > **Search** or the **TraceQL** tab.
+Both of these methods let you build queries and drill-down into result sets.
+
+To add TraceQL panels to your dashboard, refer to the [Traces panel documentation](/docs/grafana/latest/panels-visualizations/visualizations/traces/).
+
+To learn more about Grafana dashboards, refer to the [Use dashboards documentation](/docs/grafana/latest/dashboards/use-dashboards/).
 
 {{% admonition type="note" %}}
-To use the TraceQL query editor in Grafana, you need to enable the `traceqlEditor` feature flag. This feature is available starting in Grafana 9.3.2. The query editory is available automatically in Grafana Cloud.
+To use the TraceQL query editor in Grafana 9.3.2 and newer, you need to enable the `traceqlEditor` feature flag.
+
+To enable the Trace Search in Grafana 10 and newer, you need to enable the `traceqlSearch` feature flag.
```

Comment:
Comments that should not prevent from merging this PR:

1. I don't know where to edit Feature flags
2. Are we sue a feature flag must be changed? Are we sure TraceQL tab is not enabled by default in Grafana 10? I can see the TraceQL tab on my Grafana 10 on Grafana Cloud even though I don't remember tweaking my config

Reply:
I can add a link to the Feature Flag page easily enough. The feature flag isn't necessary in Grafana Cloud. i think both of these features are default in Cloud. But I will verify.

Reply:
Thank you @knylander-grafana for your pugnacity.
I think it would be great, in this doc update or a subsequent one, to be clear about the need or not of a feature flag on Grafana Cloud, especially for Grafana 10

Reply:
I agree with @cyrille-leclerc I don't think that `traceqlSearch` is required in Grafana 10.

@grafana/observability-traces-and-profiling do we have a list of what feature flags are required in which Grafana versions for the traceql search and the traceql builder replacement of old search?

Reply:
@adrapereira It's still confusing on this page that we have two panels on the same page that deal with TraceQL search. The Query type > Search is the TraceQL query builder while the TraceQL tab is the query editor. Using these terms is confusing -- especially if our own people are getting confused by the terminology. 

How do you refer to the editor and the builder?

Reply:
Here's what I did: 
![image](https://github.com/grafana/tempo/assets/104772500/e48c089f-92f2-4f62-b501-eb6945a4010c)